### ISSUE TITLE
Staff templates: adopt V1 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17066]
## Description
To be in compliance with the Design System and to help make sure our products receive design and accessibility updates, Staff templates needs to be updated to use the most recent versions of all available components.

Example pages:

- [Minneapolis Leadership List](https://www.va.gov/minneapolis-health-care/about-us/leadership/)
  - [Staff Detail page](https://www.va.gov/minneapolis-health-care/staff-profiles/amy-archer/)
- [Battle Creek Leadership List](https://www.va.gov/battle-creek-health-care/about-us/leadership/)
  - [Staff Detail page](https://www.va.gov/battle-creek-health-care/staff-profiles/michelle-martin/)

### Elements that are not using components that should be

**Leadership List template & Staff Detail template**
1. Link
   - This is for the download links and email links on the Staff Detail templates ([example](https://www.va.gov/battle-creek-health-care/staff-profiles/michelle-martin/)) and the Name links on the listing template
   - These are currently not using a component at all but they should be using [the v1 component](https://design.va.gov/components/link/#download) 
2. ~Telephone~
   - ~These are currently not using a component at all but they should be using [the v1 component](https://design.va.gov/components/telephone)~
   - ~Exts in phone numbers within the component are parsed correctly~

## Acceptance Criteria
- [ ] Link component has been added 
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I estimate this at a **2** for the same reasons as News Releases.

**Comment by laflannery** [comment]
Telephone moved to #16148

**Comment by eselkin** [comment]
Estimate a 2 as well. The links on the Leadership page is simple. The links on detail pages is slightly more complex, but should be done in conjunction with icon update.

**Comment by randimays** [comment]
Successfully validated in production. Closing!
